R/histof.R

```diff
@@ -0,0 +1,231 @@
+#' test for ABO compatibility
+#'
+#' @description ABO compatibility test between donor and candidate
+#' @param cABO A character from 'A', 'B', 'AB', 'O'
+#' @param dABO A character from 'A', 'B', 'AB', 'O'
+#' @param iso A logical value T/F
+#' @return A logical value T/F
+#' @examples
+#' abo(cABO = 'A', dABO = 'A', iso = TRUE)
+#' @export
+abo <- function(cABO = 'A', dABO = 'A', iso = TRUE){
+
+  if(iso == TRUE){
```

Comment:
We should use the check blood group function from the utility.R file to make sure the blood group is valid.